motif-docs/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "type": "module",
+  "scripts": {
+    "dev": "next dev -p 4000",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint"
+  },
+  "dependencies": {
+    "@headlessui/react": "0.0.0-insiders.ab78fbd",
```

Comment:
you need to install `sharp` so image transforms are fast

Reply:
@flybayer Do I need to do anything other than that?